EternalModLoader/EternalModLoader.cs

```diff
@@ -320,7 +414,9 @@ public static void ReplaceChunks(MemoryStream memoryStream, ResourceInfo resourc
                         }
                     }
 
-                    Console.WriteLine(string.Format("\tReplaced {0}", mod.Name));
+                    if (!mod.isBlangJson)
```

Comment:
Always use braces, even for 1 line conditions. Please follow the C# coding convention: https://docs.microsoft.com/es-es/dotnet/csharp/programming-guide/inside-a-program/coding-conventions